src/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -1624,17 +1624,19 @@ static string GetCodeAnalysisTreatWarningsAsErrorsTargetContents()
 ";
             }
 
-            static string GetPackageSpecificContents(string packageName)
-                => packageName switch
-                {
-                    CodeAnalysisAnalyzersPackageName => @"
-  <!-- Target to add all 'EmbeddedResource' files with '.resx' extension as analyzer additional files -->
+            const string AddAllResxFilesAsAdditionalFilesTarget = @"  <!-- Target to add all 'EmbeddedResource' files with '.resx' extension as analyzer additional files -->
   <Target Name=""AddAllResxFilesAsAdditionalFiles"" BeforeTargets=""GenerateMSBuildEditorConfigFileCore;CoreCompile"" Condition=""'@(EmbeddedResource)' != '' AND '$(SkipAddAllResxFilesAsAdditionalFiles)' != 'true'"">
     <ItemGroup>
       <EmbeddedResourceWithResxExtension Include=""@(EmbeddedResource)"" Condition=""'%(Extension)' == '.resx'"" />
       <AdditionalFiles Include=""@(EmbeddedResourceWithResxExtension)"" />
```

Comment:
I found [I had to add the `AdditionalFiles` items for .resx myself](https://github.com/microsoft/vs-validation/pull/177/files#diff-50e91c82311ea26f2a73202525dfdf2b0a89c178ee666b2bf3ad4c84ac4c06dcR17). Was this code not working?

Reply:
You probably don't have the very Microsoft.CodeAnalysis.NetAnalyzers installed (something after #6559).